app/helpers/passport.js

```diff
@@ -1,17 +1,20 @@
 'use strict';
+
+var request = require('request');
 var GitHubStrategy = require('passport-github').Strategy;
+var BearerStrategy = require('passport-http-bearer').Strategy;
 
 var User = require('../models/user');
+var Account = require('../models/account');
 var config = require('../../config');
 
-// expose this function to our app using module.exports
 module.exports = function(passport) {
     passport.serializeUser(function(user, done) {
```

Comment:
This file handles all the passport configuration. I pass the same instance around to all files which need authentication, so it just made sense to abstract it to a separate file to avoid re-initializing.